analysis/analysis.py

```diff
@@ -28,6 +29,87 @@ def load_data_csv(csv_path: Path):
     return out_data
 
 
+def get_sentiments() -> dict:
+    """
+    Returns a dictionary of sentiment scores, with the keys being the word and the values being
+    their score
+
+    :return: dict mapping words to their sentiment scores
+    """
+    sentiment_path = Path('data', 'sentiments.txt')
+
+    sentiments = dict()
+    with open(sentiment_path, 'r') as file:
+        word = file.readline()
+
+        # We want to handle each word individually, rather than as a whole set
+        while word is not '':
+
+            # This particular file starts lines with '#' for non-sentiment comments, so skip them
+            if word[0] == '#' or word[0] == '\t':
+                word = file.readline()
+                continue
+
+            # All words use tabs to define the different parts of the data
+            attributes = word.split('\t')
```

Comment:
After this line, can you pull all of the various attributes into their own variables before doing the stuff at 63-70? e.g., below I have no idea what ```attributes[2]``` and ```attributes[3]``` are, whereas attributes[4] is clearer since you named it.